interpreter/cling/lib/Interpreter/BackendPasses.cpp

```diff
@@ -140,9 +140,14 @@ namespace {
 
       // Find the symbol in JIT or shared libraries.
       if (m_JIT.getSymbolAddress(GV.getName(), /*AlsoInProcess*/ true)) {
+#if !defined(_WIN32)
+        // Heuristically, Windows cannot handle cross-library variables; they
+        // must be library-local.
```

Comment:
Oh, you know what this could be? Windows has this funny `dllexport` and `dllimport` thing, maybe we must not delete those regardless of whether the symbol is already there?

Reply:
We generally have issues when sharing global variables > `int` size across DLL boundary, it's not just with the JIT. That's why we typically need to outline getters for static data members, because such getters cannot be called from outside the DLL that defines the static data member. I'm not contradicting, I'm probably just giving more experimental results :-) @bellenot FYI!